src/view.cpp

```diff
@@ -359,8 +359,14 @@ void view::open_in_browser(const std::string& url) {
 		cmdline.append("'");
 	}
 	stfl::reset();
-	utils::run_interactively(cmdline, "view::open_in_browser");
+	int browser_exit_code = utils::run_interactively(cmdline, "view::open_in_browser");
+	if (browser_exit_code == 0) {
+		LOG(level::DEBUG, "Starting the browser failed with error code %d - command was %s\n", browser_exit_code, cmdline);
```

Comment:
1. No need to include "\n" at the end of the message—the logger will do that for you automatically.
2. We usually prefix our log lines with the name of the function to make it easier to follow; in this case it should be `utils::run_interactively`.